site/src/app.css

```
@@ -391,6 +397,53 @@ kbd {
+}
+.diatomic-plot-shell {
+  min-height: var(--plot-height, 300px);
+}
+.diatomic-plot-placeholder {
+  display: grid;
+  place-items: start center;
+  color: var(--text-muted);
+}
+.diatomic-plot {
+  --scatter-font-size: 14px;
+  display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)